backend/debug/dwarf/dwarf_high/operator_builder.ml

```diff
@@ -133,23 +133,42 @@ let implicit_pointer ~offset_in_bytes ~die_label dwarf_version : O.t =
 let call ~die_label ~compilation_unit_header_label : O.t =
   DW_op_call4 { label = die_label; compilation_unit_header_label }
 
+[@@@ocamlformat "wrap-comments=false"]
```

Comment:
You shouldn't need to do this, instead put an asterisk at the start of each line of the comment whose formatting is to be preserved.